### DISCUSSION TITLE
The certificate chain was issued by an authority that is not trusted

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#1801] [Q&A] [answered]
I have just downloaded the  version3.0 framework. when I change DB connection string to the existing database. got the error message:
SqlException: A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)

**Comment by leigh-pointer** [comment, marked as answer]
Hi @qunshen,
You need to update the connection string in appsettings.json to include the "Encrypt=false;" qualifier.

**Reply by qunshen** [reply]
problem solved! Thanks

**Reply by leigh-pointer** [reply]
@qunshen , can you marked this as answered please?

**Reply by mixlplix** [reply]
Trying to upgrade right now, and I am getting this error, adding Encrypt=false did not solve my problem.  Help!

Here's my connection string:

    "DefaultConnection": "Server=tcp:rds-<hidden>.amazonaws.com,1433; Initial Catalog=<hidden>; Integrated Security=false; user id=<hidden>; password=<hidden>; MultipleActiveResultSets=False; Encrypt=false;",

I tried using all combinations of TrustServerCertificate as well, no luck.

**Reply by leigh-pointer** [reply]
@mixlplix did you update the connection string in the tenants table as well?

**Reply by sbwalker** [reply]
@leigh-pointer your answer is 100% correct. This reminded me to update the release notes for version 3.0 to include information about this breaking change from Microsoft.

"In .NET 6 Microsoft introduced a breaking change in regards to SQL Server connection strings. Previously, database connections were not expected to be encrypted; however, in .NET 6 they are now expected to be encrypted by default. You can override this behavior by appending an Encrypt=False attribute to your connection string. In Oqtane, this needs to be done in both the appsettings.json file as well as the DBConnectionString column in the Tenants table in the master database. New installations of Oqtane from 3.0+ include this attribute by default. Please note that not all installations will experience this upgrade issue. It typically occurs in local development environments where you are using LocalDB or have installed SQL Server locally without using a trusted certificate."

**Comment by mixlplix** [comment]
Thank you! That’s probably it, but I won’t know until I can do the upgrade this weekend.

**Reply by mixlplix** [reply]
I can confirm that this worked

**Reply by leigh-pointer** [reply]
Great to hear
